src/main/java/net/minecraftforge/event/world/RaidEvent.java

```diff
@@ -0,0 +1,70 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+
+/**
+ *  RaidEvent along with its subevents gets fired whenever a {@link Raid} occurs.
+ */
```

Comment:
This is still inaccurate. `RaidEvent` is never fired.

Reply:
fixed that